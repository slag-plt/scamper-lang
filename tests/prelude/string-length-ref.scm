(string-length "HelloWorld")
(string-length "Hello woww World")
(string-length "")
(string-length "00110011")
(string-length "1234567")
(string-ref "HelloWorld" 7)
(string-ref "Hello woww World" 10)
(string-ref " " 0)
(string-ref "00110011" 3)
(string-ref "1234567" 5)