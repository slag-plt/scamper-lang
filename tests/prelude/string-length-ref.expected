10
16
0
8
7
#\r
#\space
#\space
#\1
#\6
